server/startup/migrations/index.js

```diff
@@ -222,4 +222,5 @@ import './v222';
 import './v223';
 import './v224';
 import './v225';
+import './v226';
```

Comment:
update

Reply:
I have now updated the migration from `v226` to `v229`.